__tests__/forces/Gravity_test.re

```diff
@@ -7,7 +7,12 @@ describe("Gravity", () => {
     it("should derive the force of gravity between two objects", () =>
       Expect.(
         expect(
-          Gravity.gravityForceMag(~attractorMass=1., ~moverMass=1., ~r=1.),
+          Gravity.gravityForceMag(
+            ~attractorMass=1.,
+            ~moverMass=1.,
+            ~r=1.,
+            (),
```

Comment:
`unit` parameter (the last `()`) has to be added to this function now that it takes an optional argument. This is how we signal that the function has been fully applied even tho we didn't supply an argument for `G`, since Reason will automatically curry functions.